lib/widgets/screens/profile/profile.dart

```diff
@@ -215,100 +227,74 @@ class _ProfileScreenScrollState extends State<ProfileScreenScroll> {
                 stage: company.companyStage?.translatedValue,
                 city: "Washington D.C.", //TODO
                 country: "USA", //TODO
-                industry: company.industries?.first,
-                expertisesSought: widget.userData.groupMemberships
-                    .firstWhere((g) => g.groupIdent == GroupIdent.mentees)
-                    .maybeWhen(
-                        menteesGroupMembership: (g) => g.soughtExpertises
-                            .map((e) => e.translatedValue!)
-                            .toList(),
-                        orElse: () => []),
+                industry: maybeMenteeGroupMembership?.industry?.translatedValue,
+                expertisesSought: maybeMenteeGroupMembership?.soughtExpertises
+                        .map((e) => e.translatedValue!)
+                        .toList() ??
+                    [],
                 mission: company.description,
                 imageUrls: [
                   "https://st2.depositphotos.com/1326558/7163/i/600/depositphotos_71632883-stock-photo-mexican-tacos-with-meat-vegetables.jpg",
                   "https://st3.depositphotos.com/13349494/32631/i/600/depositphotos_326317470-stock-photo-cropped-view-man-adding-minced.jpg",
-                ],
-                motivation:
-                    'Sit amet justo donec enim diam vulputate ut pharetra sit amet aliquam id diam maecenas ultricies. Mi eget mauris pharetra et ultrices neque ornare aenean euismod elementum nisi quis eleifend.', //TODO
+                ], //TODO
+                motivation: //TODO
```

Comment:
`motivationsForMentorship` is on the MenteesGroupMembership

Reply:
Added!